MAVProxy/modules/mavproxy_map/mp_tile.py

```diff
@@ -35,10 +35,13 @@
     from urllib2 import Request as url_request
     from urllib2 import urlopen as url_open
     from urllib2 import URLError as url_error
+    download_error = url_error
 else:
     from urllib.request import Request as url_request
     from urllib.request import urlopen as url_open
     from urllib.error import URLError as url_error
+    from http.client import RemoteDisconnected
+    download_error = (RemoteDisconnected, url_error)
```

Comment:
```suggestion
    from urllib.error import URLError as actual_url_error
    from http.client import RemoteDisconnected
    url_error = (RemoteDisconnected, actual_url_error)
```

Reply:
Get it , let me change it with better name